gram.nim

```diff
@@ -4,6 +4,9 @@ import std/intsets
 import std/macros
 import std/hashes
 import std/sets
+import std/options
+import hasts/graphviz_ast
+export topng, toDotNodeId
```

Comment:
Are you sure you want to export these directly?  I think we probably want to wrap at least a little in order to assert API, document them in gram, etc.